packages/py-api/py_api/models/hackathon_teams_models.py

```diff
@@ -0,0 +1,19 @@
+from typing import List, Literal, Optional
+
+from pydantic import BaseModel
+
+# normal team is a team where the admin sends link
```

Comment:
Elaborate a bit. Please, use chat gpt to generate more descriptive and well structured sentences.